src/main/java/org/seqdoop/hadoop_bam/KeyIgnoringVCFOutputFormat.java

```diff
@@ -90,7 +95,23 @@ public void readHeaderFrom(SeekableStream in) throws IOException {
 			TaskAttemptContext ctx)
 		throws IOException
 	{
-		return getRecordWriter(ctx, getDefaultWorkFile(ctx, ""));
+		Configuration conf = ctx.getConfiguration();
+		boolean isCompressed = getCompressOutput(ctx);
+		CompressionCodec codec = null;
+		String extension = "";
+		if (isCompressed) {
+			Class<? extends CompressionCodec> codecClass =
+					getOutputCompressorClass(ctx, GzipCodec.class);
```

Comment:
GzipCodec seems like an unnatural default, certainly for BCF. Wouldn't BGZFCodec be a better choice for both for both VCF and BCF?